constants.py

```diff
@@ -77,4 +77,5 @@
          14:"يمنع إرسال روابط لسيرفرات أخرى للأعضاء على الخاص، إذا تم التبليغ أكثر من مرة على نفس العضو يعطى إنذار",
          15:" السؤال والإستفسار والمجادلة لقرارات المشرفين عرضة للحصول على إنذار مباشر مثل السؤال عن سبب عقابه عضو آخر",
          16:" يمنع إستخدام أي صورة رمزية خادشة للحياء، كما يمنع إستخدام أسماء مخالفة لذلك، كل إسم أو صورة رمزية مخلة بالآداب أو تدعو للكراهية والعنصرية سيتم تنبيه صاحبها لتعديلها كحد أقصى 24 ساعة، فإذا لم يستجب سنضطر آسفين لحظره",
-         77:"حاول ان تحب البطاطا!"}
+         77:"حاول ان تحب البطاطا!"
+         0:"المرجوا عدم عمل منشن عشواءيا للاعضاء"}
```

Comment:
1. rules start at 1
2. this rule does not exists in the server, we "admins" needs to agree on it first, so we can add it